lib/localparticipant.js

```diff
@@ -143,8 +143,8 @@ LocalParticipant.prototype._handleTrackSignalingEvents = function _handleTrackSi
   var signaling = this._signaling;
 
   function localTrackAdded(localTrack) {
-    var mediaStreamTrackOrDataTrackSender = localTrack._dataTrackSender || localTrack.mediaStreamTrack;
-    signaling.addTrack(mediaStreamTrackOrDataTrackSender, localTrack.name);
+    var trackSender = localTrack._dataTrackSender || localTrack._mediaTrackSender;
```

Comment:
Just wondering: would we benefit from letting this property be `_trackSender` in both cases instead of `_dataTrackSender` and `_mediaTrackSender`?